Jint/Runtime/Interop/ClrHelper.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Jint.Native;
+
+namespace Jint.Runtime.Interop
+{
+    public static class ClrHelper
```

Comment:
this can probably be internal? see also earlier comment about being instantiable and having `allowGetType` as constructor parameter

Reply:
I will make the class public but with an internal constructor.

Reply:
I have made `ClrHelper` having `InteropOptions` as constructor parameter, to make it possible to change `Engine.Options.Interop.AllowGetType` after Engine created.